### ISSUE TITLE
open with

### ISSUE THREAD

**Issue by yephny** [AmanoTeam/UnalixAndroid#1]
I think Unalix is necessary for any device. Though, I'd love it if the app had open with functionality. Unalix is practically equivalent to [link eye](https://www.f-droid.org/packages/kuesji.link_eye/), but link eye is discontinued. What do you think?

**Comment by SnwMds** [comment]
> What do you think?

Well, I don't care.

**Comment by yephny** [comment]
I see that I have worded my question poorly, I did not mean to focus on another app, that was simply an example. I am sorry for causing confusion. @SnwMds 

What I meant to say was that Unalix is a great app (also has the cleanest UI among the similar apps I've used) and it even has the "open with" feature that I am talking about; so, it looks like everything is already implemented. The feature that I am requesting here is for the app to be listed in the Open with menu, though only with a single action instead of three (clean, copy to clipboard, unshort). 

This means that the user would click a link, select Unalix (or set it to always open with Unalix, so that there would not even be a need for an extra step), and see the link they clicked is already pasted into Unalix. Now the user would be able to use the buttons to clean, unshort, share, and copy the url. What would even be better is if all apps compatible with the link are listed in Unalix, so that the user can open the link with the app they want with a single click. This would, I guess, call for removing the clear text box button (X), and replacing it with a copy button. The copy functionality could also be implemented as a feature that would work when you hold down one of the buttons, like the share button.
